bin/pyntlm_auth/app.py

```diff
@@ -0,0 +1,272 @@
+import binascii
+import configparser
+import datetime
+import os
+import socket
+import sys
+import threading
+from configparser import ConfigParser
+from http import HTTPStatus
+
+from flask import Flask, request
+from samba import param, NTSTATUSError, ntstatus
+from samba.credentials import Credentials, DONT_USE_KERBEROS
+from samba.dcerpc import netlogon
+from samba.dcerpc.misc import SEC_CHAN_WKSTA
+from samba.dcerpc.netlogon import (netr_Authenticator)
+
+machine_cred = None
+secure_channel_connection = None
+connection_id = 1
+reconnect_id = 0
+connection_last_active_time = datetime.datetime.now()
+lock = threading.Lock()
+
+conf_path = "/usr/local/pf/conf/domain.conf"
+listen_port = os.getenv("LISTEN")
+identifier = os.getenv("IDENTIFIER")
```

Comment:
check that both of these variables exist in the environment, and raise an error and exit if missing.

...or instead - bake in defaults for when they're undefined